lib/rules/max-line-length/__tests__/index.js

```diff
@@ -238,7 +238,7 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
-  config: [ 20, { ignorePattern: "/https?:\/\/[0-9,a-z]*.*/" } ],
+  config: [ 20, { ignorePattern: "/https?://[0-9,a-z]*.*/" } ],
```

Comment:
Per the rule docs though this should be a regex though should it not?
https://stylelint.io/user-guide/rules/max-line-length/#ignorepattern-regex

I can't remember which rule or rules have the option to use a _string_ or a _regex_ but it doesn't appear that `max-line-length` is one of the rules that allows a _string_ or _regex_ in the pattern, only a _regex_ based on the rule documentation